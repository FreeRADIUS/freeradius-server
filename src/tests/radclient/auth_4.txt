#
#	ARGV: -i 123 -c 1 -x -F
#
User-Name = "bob",
User-Password = "hello"
NAS-Identifier = "auth_4"
