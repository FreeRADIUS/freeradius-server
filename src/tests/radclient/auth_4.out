Sent Access-Request Id 123 from 0.0.0.0:1234 to 127.0.0.1:12340 length 51 
        Password.Cleartext = "hello"
        User-Name = "bob"
        User-Password = "hello"
        NAS-Identifier = "auth_4"
Received Access-Accept Id 123 from 127.0.0.1:12340 to 0.0.0.0:1234 via lo length 47 
        Class = 0x483d342c493d34
        raw.Vendor-Specific = 0x483d342c493d34
        raw.Vendor-Specific = 0x483d342c493d43
